Makefile

```diff
@@ -181,7 +181,7 @@ helm.docs: ## Generate helm docs
 	@cd $(HELM_DIR); \
 	$(DOCKER) run --rm -v $(shell pwd)/$(HELM_DIR):/helm-docs -u $(shell id -u) docker.io/jnorwood/helm-docs:v1.7.0
 
-HELM_VERSION ?= $(shell helm show chart $(HELM_DIR) | grep 'version:' | sed 's/version: //g')
+HELM_VERSION ?= $(shell helm show chart $(HELM_DIR) | grep '^version:' | sed 's/version: //g')
```

Comment:
This was required because, version of the dependency package (bitwarden-sdk-server) was also included in the command output and was causing `make helm.build` to fail.

Output without the change:
```
$ helm show chart deploy/charts/external-secrets | grep 'version:' | sed 's/version: //g'
  v0.5.0
0.18.2
```